index.js

```diff
@@ -46,17 +44,14 @@ module.exports = robot => {
     const {owner, repo} = context.repo();
     const q = `label:"${freeze.config.labelName}" repo:${owner}/${repo}`;
 
-    context.github.search.issues({q}).then(resp => {
-      resp.data.items.forEach(issue => {
-        context.github.issues.getComments(githubHelper.parseCommentURL(issue.comments_url)).then(resp => {
-          return freeze.getLastFreeze(resp.data);
-        }).then(lastFreezeComment => {
-          if (freeze.unfreezable(lastFreezeComment)) {
-            freeze.unfreeze(issue, formatParser.propFromComment(lastFreezeComment));
-          }
-        });
-      });
-    });
-    console.log('scheduled thaw run complete');
+    const resp = await context.github.search.issues({q});
+
+    await Promise.all(resp.data.items.map(issue => {
+      // Issue objects from the API don't include owner/repo params, so
+      // setting them here with `context.repo` so we don't have to worry
+      // about it later. :/
```

Comment:
isn't that weird? That's why i wrote the url parser to start with.  Not sure if this is good/bad/something that we should advocate for upstream, but seems odd to me.